pkg/sysinfo/sysinfo.go

```diff
@@ -3,23 +3,42 @@ package sysinfo
 // SysInfo stores information about which features a kernel supports.
 // TODO Windows: Factor out platform specific capabilities.
 type SysInfo struct {
+	// Whether support APpArmor or not
 	AppArmor bool
 	*cgroupMemInfo
-	*cgroupCpuInfo
-	IPv4ForwardingDisabled        bool
-	BridgeNfCallIptablesDisabled  bool
-	BridgeNfCallIp6tablesDisabled bool
-	CgroupDevicesEnabled          bool
+	*cgroupCPUInfo
+
+	// If IPv4 forwarding was disabled, networking will not work
```

Comment:
You should choose between "if" and "wether" :stuck_out_tongue_closed_eyes: :wink:.